File: app/src/main/java/com/fahamutech/doctorapp/database/connector/ArticleDataSource.java
package com.fahamutech.doctorapp.database.connector;

import androidx.swiperefreshlayout.widget.SwipeRefreshLayout;
import androidx.recyclerview.widget.RecyclerView;

public interface ArticleDataSource {
    void getArticles(String categoryId, RecyclerView recyclerView,
                     SwipeRefreshLayout swipeRefreshLayout);
}

File: core/src/main/java/com/fahamutech/kcore/user/database/connector/ArticleDataSource.kt
package com.fahamutech.kcore.user.database.connector;

import androidx.recyclerview.widget.RecyclerView;
import androidx.swiperefreshlayout.widget.SwipeRefreshLayout;

public interface ArticleDataSource {
    void getArticles(String categoryId, RecyclerView recyclerView,
                     SwipeRefreshLayout swipeRefreshLayout);
}
